src/components/track-changes/suggestion-list/SuggestionSnippet.tsx

```diff
@@ -10,28 +10,26 @@
  * All portions of the code written by Atypon Systems LLC are Copyright (c) 2024 Atypon Systems LLC. All Rights Reserved.
  */
 import {
-  CHANGE_OPERATION,
   CHANGE_STATUS,
   ChangeSet,
   TrackedChange,
 } from '@manuscripts/track-changes-plugin'
-import { schema } from '@manuscripts/transform'
-import parse from 'html-react-parser'
 import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
 
 import {
-  getFootnoteText,
-  getInlineFootnoteContent,
-} from '../../../lib/footnotes'
-import { changeOperationAlias } from '../../../lib/tracking'
-import { getParentNode } from '../../../lib/utils'
+  handleNodeChange,
+  handleTextChange,
+  handleUnknownChange,
+} from '../../../lib/change-handlers'
 import { useStore } from '../../../store'
+import SnippetContent from './SnippetContent'
 
 interface SnippetData {
   operation: string
   nodeName: string
   content: string
+  isEquation?: boolean
```

Comment:
Can't you just use `nodeName` to check if it is an equation?

Reply:
Yes, it's possible. However, implementing it this way will require more logic and extra code. I believe the current approach is much shorter and serves the purpose effectively.